tests/test_flow/test_executor.py

```diff
@@ -114,7 +114,7 @@ def test_parallel_fail(aip_builder, make_counter, multiprocessing_manager, log_c
 
     # The barrier ensures that all the entity functions do not complete unless
     # all of them are started.
-    barrier = multiprocessing_manager.Barrier(4, timeout=120)
+    barrier = multiprocessing_manager.Barrier(4, timeout=240)
```

Comment:
From our conversation, it sounds like we only have a high timeout for debugging purposes, and we should never hit this normally. Maybe we can add a comment explaining this?